templates/bootstrap/setup.py.j2

```diff
@@ -9,11 +9,13 @@ setup(
     name="{{ plugin_name | dash }}",
     version="0.1.0a1.dev",
     description="{{ plugin_name | dash }} plugin for the Pulp Project",
+    long_description="{{ plugin_name | dash }} plugin for the Pulp Project",
+    long_description_content_type="text/markdown",
     license="GPLv2+",
     author="AUTHOR",
     author_email="author@email.here",
     url="http://example.com/",
-    python_requires=">=3.8",
+    python_requires=">=3.9",
```

Comment:
I think this would be applied to old branches?  Do we need a new template variable for `minimum_supported_python`?

Reply:
Once bootstrapped, this is a manual change. I don't think it's worth a variable.